kubernetes-client/src/main/java/io/fabric8/kubernetes/client/dsl/base/HasMetadataOperation.java

```diff
@@ -78,8 +78,10 @@ public T replace(T item) {
         if (got == null) {
           return null;
         }
+
+        item = handleReplace(item);
```

Comment:
-ETOOMUCHJAVA for me :)

I could study all this stuff and try to make something, but if you have the ability, time and knowledge to make that (100x faster than me BTW), please, go ahead!